pkg/process/checks/net.go

```diff
@@ -119,115 +123,21 @@ func (c *ConnectionsCheck) getConnections() ([]ebpf.ConnectionStats, error) {
 	return tu.GetConnections(c.tracerClientID)
 }
 
-// Connections are split up into a chunks of at most 100 connections per message to
-// limit the message size on intake.
-func (c *ConnectionsCheck) formatConnections(conns []ebpf.ConnectionStats) []*model.Connection {
+func (c *ConnectionsCheck) formatConnections(conns []*model.Connection) []*model.Connection {
```

Comment:
nit: let's rename this to enrichConections or something simila since it's not doing any "formatting"

Reply:
👍